app/jobs/generate_occupation_standard_pdf_job.rb

```diff
@@ -4,12 +4,7 @@ class GenerateOccupationStandardPdfJob < ApplicationJob
   def perform(occupation_standard_id, force: false)
     os = OccupationStandard.find(occupation_standard_id)
     if force || os.should_generate_attachment?('pdf')
-      pdf = ::OccupationStandardPdf.new(os)
-      os.pdf.attach(
-        io: StringIO.new(pdf.render),
-        filename: "#{os.export_filename}.pdf",
-        content_type: "application/pdf",
-      )
+      os.generate_pdf!
```

Comment:
Nice fix, probably don't need my force flag anymore since easier to run from console. Thanks for fixing my pdf attempt :)

Reply:
@littleforest wasnt a fix as much as a personal convenience so I could more quickly iterate from the console to check newly generated PDFs without having everything on the web working!